README.md

```diff
@@ -7,18 +7,58 @@ The code here includes regression test data of computation units coming from ser
 
 **WARNING** This repo is under active development and relies on code and data that is not publicly available at this point.
 
-## Getting started
+## QuickStart
 
-Use the `tests` target of the Makefile to run the unit tests.
+1. Ensure you have docker installed and available for building and running and has access to the VCM cloud
+
+You should enable running docker as your user rather than as root with sudo. First, create a docker group if it doesn't exist:
+
+```shell
+$ sudo groupadd docker
+```
+
+Add your user to this group:
+
+```shell
+$ sudo usermod -aG docker $USER
+```
+
+Logout/restart to ensure these changes take effect.
+
+Finally, authorize docker to pull from GCP:
+
+```shell
+$ gcloud auth configure-docker
+```
+
+2. To build the image, download the data, and run the tests, your first step should be:
 
 ```shell
 $ make tests
 ```
-This will pull the test data from the Google storage bucket (using the `make get_test_data`) if it does not exist locally yet, build the fv3core docker image, and run all of the sequential tests using that image.
 
-See the [Unit Testing]() section below for options.
+If you want to develop code, you should also install the linting requirements locally
+
+```shell
+$ pip install -c constraints.txt -r requirements_lint.txt
+```
```

Comment:
The default suggestion to new users should be to install pre-commit, since it reduces the burden on new users to learn active commands they need to run as part of their development workflow.

```suggestion
If you want to develop code, you should also install the linting requirements and git hooks locally

```shell
$ pip install -c constraints.txt -r requirements_lint.txt
$ pre-commit install
```
```